android/clientlib-ktx/build.gradle

```diff
@@ -33,6 +35,84 @@ android {
     buildFeatures {
         buildConfig = false
     }
+
+    publishing {
+        singleVariant('release') {
+            withSourcesJar()
+            withJavadocJar()
+        }
+    }
+}
+
+publishing {
+    publications {
+        release(MavenPublication) {
+            groupId = group
+            artifactId = 'mobile-wallet-adapter-clientlib-ktx'
```

Comment:
Along with `groupId = group` and `artifactId = 'mobile-wallet-adapter-clientlib-ktx'`, we might want to add `version = version + '-alpha'`, unless we are sufficiently satisfied with the API surface that we are willing to commit to it. @creativedrewy thoughts?

Reply:
Not needed since this is already used in a couple of sample apps although not as a dependency